packages/components/src/Chip/Chip.tsx

```diff
@@ -9,56 +9,46 @@ import { useChildComponent } from "./hooks/useChildComponent";
 import { Typography } from "../Typography";
 import { Tooltip } from "../Tooltip";
 
-export const Chip = ({
-  ariaLabel,
-  disabled,
-  heading,
-  invalid,
-  label,
-  value,
-  testID,
-  onClick,
-  onKeyDown,
-  children,
-  role = "button",
-  tabIndex = 0,
-  variation = "base",
-}: ChipProps): JSX.Element => {
-  const classes = classnames(styles.chip, {
-    [styles.invalid]: invalid,
-    [styles.base]: variation === "base",
-    [styles.subtle]: variation === "subtle",
-    [styles.disabled]: disabled,
-  });
+const ChipComponent = forwardRef<HTMLButtonElement | HTMLDivElement, ChipProps>(
+  (
+    {
+      ariaLabel,
+      disabled,
+      heading,
+      invalid,
+      label,
+      value,
+      testID,
+      onClick,
+      onKeyDown,
+      children,
+      role = "button",
+      tabIndex = 0,
+      variation = "base",
+    },
+    ref,
+  ) => {
+    const classes = classnames(styles.chip, {
+      [styles.invalid]: invalid,
+      [styles.base]: variation === "base",
+      [styles.subtle]: variation === "subtle",
+      [styles.disabled]: disabled,
+    });
 
-  const prefix = useChildComponent(children, d => d.type === Chip.Prefix);
-  const suffix = useChildComponent(children, d => d.type === Chip.Suffix);
+    const prefix = useChildComponent(children, d => d.type === ChipPrefix);
+    const suffix = useChildComponent(children, d => d.type === ChipSuffix);
```

Comment:
this should be equivalent since `Chip.Prefix` was literally `ChipPrefix` before, however now with the change in how we're setting up the namespace we can't yet reference Chip.Prefix.